src/scripts/main.js

```diff
@@ -5,4 +5,26 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+for (let i = 0; i < people.length; i++) {
+  const tableRow = document.createElement('tr');
+  const age = people[i].died - people[i].born;
+  const century = Math.ceil(people[i].died / 100);
+  let gender = 'Male';
+
+  if (people[i].sex === 'f') {
+    gender = 'Female';
+  };
```

Comment:
```suggestion
  let gender = people[i].sex === 'f' 
    ? 'Female' 
    : 'Male';
```
Тут доречніше було б використати тернарник

Reply:
ОК. Дякую